website/docs/getting-started/configure.md

```diff
@@ -4,4 +4,52 @@ sidebar_position: 6
 ---
 # Configuring odo global settings
 
-This doc explains everything in `~/.odo/preference.yaml` file.
\ No newline at end of file
+The global settings for odo can be found in `preference.yaml` file; which is located by default in the `.odo` directory of the user's HOME directory.
```

Comment:
Can we override these global settings per component?
What if I want to set BuildTimeout to 300 seconds for one component, but 200 for another, is it possible?